src/Sfx/App/Scripts/Services/StatusWarningService.ts

```diff
@@ -0,0 +1,73 @@
+module Sfx {
+
+    export interface IStatusWarning {
+        message: string;
+        link?: string;
+        linkText?: string;
+        level: string;
+        priority: number;
+        id: string;
+    }
+
+    export class StatusWarningService {
+        notifications: IStatusWarning[] = [];
+        storage: StorageService;
+
+        constructor(storage: StorageService) {
+            console.log(storage);
```

Comment:
Do we want to actually log this? If so, should we be logging some context?

Reply:
this shouldnt be logged.